src/cargo/ops/cargo_rustc/mod.rs

```diff
@@ -63,89 +65,97 @@ pub fn compile_targets<'a, 'cfg: 'a>(targets: &[(&'a Target, &'a Profile)],
+            let kind = Kind::from(target);
+            for filename in try!(cx.target_filenames(pkg, target, profile,
+                                                     kind)).iter() {
+                let dst = cx.out_dir(pkg, kind, target).join(filename);
+                if profile.test {
+                    tests.push((target.name().to_string(), dst));
+                } else if target.is_bin() || target.is_example() {
+                    cx.compilation.binaries.push(dst);
+                } else if target.is_lib() {
+                    let pkgid = pkg.package_id().clone();
+                    cx.compilation.libraries.entry(pkgid).or_insert(Vec::new())
+                      .push((target.clone(), dst));
+                }
+                if !target.is_lib() { continue }
 
-    for &(target, profile) in targets {
-        let kind = Kind::from(target);
-        for filename in try!(cx.target_filenames(pkg, target, profile,
-                                                 kind)).iter() {
-            let dst = cx.out_dir(pkg, kind, target).join(filename);
-            if profile.test {
-                cx.compilation.tests.push((target.name().to_string(), dst));
-            } else if target.is_bin() || target.is_example() {
-                cx.compilation.binaries.push(dst);
-            } else if target.is_lib() {
-                let pkgid = pkg.package_id().clone();
-                cx.compilation.libraries.entry(pkgid).or_insert(Vec::new())
-                  .push((target.clone(), dst));
-            }
-            if !target.is_lib() { continue }
+                // Include immediate lib deps as well
+                for dep in &cx.dep_targets(pkg, target, kind, profile) {
+                    let (pkg, target, profile) = *dep;
+                    let pkgid = pkg.package_id();
+                    if !target.is_lib() { continue }
+                    if profile.doc { continue }
+                    if cx.compilation.libraries.contains_key(&pkgid) {
+                        continue
+                    }
 
-            // Include immediate lib deps as well
-            for dep in cx.dep_targets(pkg, target, kind, profile) {
-                let (pkg, target, profile) = dep;
-                let pkgid = pkg.package_id();
-                if !target.is_lib() { continue }
-                if profile.doc { continue }
-                if cx.compilation.libraries.contains_key(&pkgid) { continue }
-
-                let kind = kind.for_target(target);
-                let v = try!(cx.target_filenames(pkg, target, profile, kind));
-                let v = v.into_iter().map(|f| {
-                    (target.clone(), cx.out_dir(pkg, kind, target).join(f))
-                }).collect::<Vec<_>>();
-                cx.compilation.libraries.insert(pkgid.clone(), v);
+                    let kind = kind.for_target(target);
+                    let v =
+                        try!(cx.target_filenames(pkg, target, profile, kind));
```

Comment:
Line breaking in Cargo typically happens on arguments instead of after `=` signs like this